AdyenDropIn/DropInComponentExtensions.swift

```diff
@@ -122,7 +122,11 @@ extension DropInComponent: FinalizableComponent {
 
     public func didFinalize(with success: Bool, completion: (() -> Void)?) {
         stopLoading()
-        selectedPaymentComponent?.finalizeIfNeeded(with: success, completion: completion)
+        if let selectedPaymentComponent = selectedPaymentComponent {
+            selectedPaymentComponent.finalizeIfNeeded(with: success, completion: completion)
+        } else {
+            completion?()
+        }
```

Comment:
This is good catch..
I am wandering how can we make it bit more simple for integration 🤔

Maybe we can make `finalizeIfNeeded ` extension for `Optional<Component>` ?

Reply:
i don't think it will help much, because in order for the extension functions to work, merchants must not unwrap the optional with a "?" otherwise the inner component functions will be invoked and if the component is nil then completion will never be called

Reply:
yep, it's not obvious helper 🤔